docs/reference/search/search-your-data/semantic-search-inference.asciidoc

```diff
@@ -40,17 +39,15 @@ include::{es-ref-dir}/tab-widgets/inference-api/infer-api-task-widget.asciidoc[]
 ==== Create the index mapping
 
 The mapping of the destination index - the index that contains the embeddings that the model will create based on your input text - must be created.
-The destination index must have a field with the <<dense-vector, `dense_vector`>>
-field type to index the output of the used model.
+The destination index must have a field with the <<dense-vector, `dense_vector`>> field type for most models and the <<sparse-vector, `sparse_vector`>> field type for the `elser` service to index the output of the used model.
```

Comment:
Nit: Sparse vector can be used with any sparse embeddings, not just ELSER.

Reply:
I made the text clearer. Our only service that uses sparse vectors currently is ELSER, and I wanted to highlight it in the tutorial so people don't need to ponder if they use the correct field type or not.